packages/ra-ui-materialui/src/layout/AppBar.js

```diff
@@ -47,12 +47,39 @@ const useStyles = makeStyles(
     }),
     { name: 'RaAppBar' }
 );
-
+/**
+ * The AppBar component renders a custom MuiAppBar.
+ *
+ * @param {Function} children Must be a function which will be called with the mutate callback
```

Comment:
No, this is wrong. Children should be a React node.

Reply:
Ok. I've copied it from another ra components actually.